microovn/api/services/control.go

```diff
@@ -32,7 +33,15 @@ func enableService(s *state.State, r *http.Request) response.Response {
 	if err != nil {
 		return response.InternalError(err)
 	}
-	return response.SyncResponse(true, requestedService+" enabled")
+
+	scr := types.ServiceControlResponse{}
+	scr.Warnings, err = node.ServiceWarnings(s)
+	if err != nil {
+		return response.InternalError(err)
```

Comment:
A good defensive practice in APIs is to not expose cause of internal server error to the caller. The error should be logged instead (preferably with some added context) and API caller should receive only generic error message. Example:
https://github.com/canonical/microovn/blob/803dc03ea847d214941330c74cc0ae0027550dc0/microovn/api/certificates/issue.go#L31-L34